KomaMRICore/src/simulation/SimulatorCore.jl

```diff
@@ -262,7 +262,7 @@ function simulate_slice_profile(
     seq::Sequence;
     z=range(-2.e-2, 2.e-2, 200), sim_params=Dict{String,Any}("Δt_rf" => 1e-6)
 )
-    sim_params["return_type"] = SpinsStateSimOutput()
```

Comment:
This should not be like this in master, please check if there is any other change

Reply:
That bug should be the last of its kind. Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/248/commits/7e191c1ee9c0decfe6be83102438be868ad88e8a